src/checklist/checklist.py

```diff
@@ -0,0 +1,43 @@
+import os
+from typing import Union
+
+import fire
+from ruamel.yaml import YAML
+
+
+class Checklist:
```

Comment:
recommend adding function 'get_all_tests' and 'get_test_areas'

Reply:
Added the above methods in the latest commit.